lib/postal/http.rb

```diff
@@ -15,7 +15,7 @@ def self.post(url, options = {})
     def self.request(method, url, options = {})
       options[:headers] ||= {}
       uri = URI.parse(url)
-      request = method.new(uri.path.length == 0 ? "/" : uri.path)
+      request = method.new((uri.path.length == 0 ? "/" : uri.path) + (uri.query ? "?" + uri.query : ""))
```

Comment:
Fixed bug here from initial pull request and squashed it back into one commit. We noticed webhooks that contained query parameters were working, but those that were not were not getting sent out whatsoever.

The issue here was that, on line 17 when the url string gets parsed into an object, url.query does not exist unless there was query parameters in the url string. Previously, the ternary operator was checking if uri.query.length == 0, since uri.query did not exist, the following error was being thrown:

`NoMethodError: undefined method `length' for nil:NilClass`